apps/web/pages/v2/settings/developer/api-keys.tsx

```diff
@@ -45,10 +45,10 @@ const ApiKeysView = () => {
         <>
           {isLoading && <SkeletonLoader />}
           <div>
-            {data?.length ? (
+            {isLoading && data === undefined ? null : !isLoading && data && data.length > 0 ? (
```

Comment:
```suggestion
            {isLoading ? null : data?.length ? (
```

Reply:
It can be simplified as above.

Reply:
that's better, i'll push the requested changes.